src/rf/rf/urls.py

```diff
@@ -19,8 +20,11 @@
 urlpatterns = patterns(
     '',
     url(r'^', include(apps.home.urls)),
+    url(r'^', include(apps.core.urls)),
```

Comment:
The idea behind `core` is that it doesn't contain any views or urls. Instead, it contains common utilities and models for all other apps in the project. We should consider merging `core` and `home` if the difference between them is too ambiguous.